README.md

```diff
@@ -341,6 +342,24 @@ The Puppet agent on a Windows node can run DSC with a normal default install. If
 * The puppet types are built from the source code of each DSC Resources MOF schema files.
 * If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](README_BUILD.md)
 
+## Places to Learn More About DSC
+
+You can learn more about PowerShell DSC from the following online resources:
+
+- [TechNet Microsoft PowerShell Desired State Configuration Overview](https://technet.microsoft.com/en-us/library/dn249912.aspx)
+- [TechNet Microsoft PowerShell DSC Resources page](https://technet.microsoft.com/en-us/library/dn249921.aspx) - For more information about built-in DSC Resources
+- [Microsoft PowerShell xDSCResources Github Repo](https://github.com/PowerShell/DscResources) -  For more information about xDsc Resources
+- Windows PowerShell [Blog](http://blogs.msdn.com/b/powershell/archive/tags/dsc/)
+- Ethan Brown's Puppet and DSC [talk](https://github.com/Iristyle/puppetconf2015)
```

Comment:
So those are the examples for my talk, so might want to slightly reword that.

My talk is actually at https://www.youtube.com/watch?v=TP0zqe-yQto - I think we plan on promoting it officially, but I don't think it hurts to post here as well if you think that's OK.

@alannab is it cool to link to the prior webinar at https://puppetlabs.com/webinars/windows-dsc-wsus-webinar-09-17-2015 ?